README.md

```diff
@@ -114,6 +151,8 @@ runs: `^<TEST_SUITE_NAME>$` where `^` signifies the start of a string and `$`
 signifies the end of a string. For example, use `riot -v run -s -x ^redis$` to
 run only the redis suite.
 
+##### tox
```

Comment:
I'm guessing this section is WIP, right?

Reply:
yes, and hopefully will go away at some point. we only have a few tests still running on `tox`.